### PR TITLE
add simple-line-icons v2.0.0 with git autoupdate, close #5439

### LINKED ISSUE #5439: [Request] Add Simple Line Icons Webfont
**Library name:**   Simple Line Icons Webfont
**Git repository url:** http://graphicburger.com/simple-line-icons-webfont/
**npm package url(optional):** 
**Official homepage:** http://graphicburger.com/simple-line-icons-webfont/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/thesabbir/simple-line-icons
**Watch 6
  Star 113
 Fork 28**
@Amomo could u help me check this PR for #5439? thank you!
i have added some keywords from `bower.json`
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is co

### CODE REVIEW

**Comment by Amomo** [comment]
@maruilian11 If you want to add Web-fonts files, I think you also need to add `.eot`, `.ttf`, `.woff` to provide compatibility. They are also be used in css file.

File: ajax/libs/simple-line-icons/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  "keywords": [
+    "icon",
+    "simple",
+    "line",
+    "css",
+    "webfont",
+    "fonts",
+    "simple-line",
+    "icon"
```
Comment (Amomo) [inline ajax/libs/simple-line-icons/package.json]: You add this keyword already~

Comment (maruilian11) [inline ajax/libs/simple-line-icons/package.json]: haha bad eyes~
thank you :D

**Comment by maruilian11** [comment]
@Amomo i have corrected it! could u help me check again? thank you!!

File: ajax/libs/simple-line-icons/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "author": "Sabbir Ahmed",
+  "license": "MIT",
+  "homepage": "https://github.com/thesabbir/simple-line-icons",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/thesabbir/simple-line-icons.git",
+    "basePath": "",
+    "files": [
+      "css/simple-line-icons.css",
+      "fonts/Simple-Line-Icons.+(svg||eot||ttf||woff)"
```
Comment (Amomo) [inline ajax/libs/simple-line-icons/package.json]: You should use `|` in regular expression.

**Comment by maruilian11** [comment]
@Amomo haha but i try them in terminal,they work, too!XDD
i will rebase it later~

**Comment by maruilian11** [comment]
@Amomo i have corrected it~ could u help me check it again? thank you~

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look? Thank you~

**Comment by PeterDaveHello** [comment]
Let's add the minified file in the `requiredFiles` array.

**Comment by Amomo** [comment]
@PeterDaveHello `simple-line-icons.min.css` has pointed in filename field, do you mean it still need to add in requiredFiles array?

**Comment by PeterDaveHello** [comment]
oh, okay, my fault.

**Comment by PeterDaveHello** [comment]
What's the reason why to add an old version like `v2.0.0` @maruilian11 @Amomo ?

**Comment by Amomo** [comment]
Oops, a newer version was released on yesterday!
Sorry I didn't notice that.

**Pull request merged**